docs/maps/search.asciidoc

```diff
@@ -2,10 +2,12 @@
 [[maps-search]]
 == Search geographic data
 
-Use the filters, query bar, and time filter to focus in on just the data you want.
-Only layers requesting data from {es} are narrowed when you submit a search request.
-Layers narrowed by the filters and query bar contain the filter icon image:maps/images/filter_icon.png[] next to the layer name in the legend.
-Only layers requesting data from {es} using an <<index-patterns, index pattern>> with a configured time field are narrowed by the time filter.
+Search across your Elasticsearch layers to focus in on just the data you want.
```

Comment:
Is the term "Elasticsearch layers" the right term?

Reply:
How about just "layers"?

Reply:
Or "the layers in your map"?